magento/magento2
# Fix date timestamp typo ( Created and Modified date are different for newly saved Image Preview)

### Description (*)


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes https://github.com/magento/adobe-stock-integration/issues/1708: Created and Modified date are different for newly saved Image Preview

### Manual testing scenarios (*)
1. Go to **Content - Media Gallery**
2. Click **Search Adobe Stock** and Save an image Preview
3. Select the previously saved preview image
4. Click "three dots", select **View Details**
5. Observe the _Created_ and _Modified_ dates

The _Created_ and _Modified_ dates should be equal


### Questions or comments


### Contribution checklist (*)
